WeakAuras/GenericTrigger.lua

```diff
@@ -1358,17 +1359,15 @@ function GenericTrigger.Add(data, region)
                   trigger_unit_events[i] = trigger_unit_events[i] or {}
                   tinsert(trigger_unit_events[i], trueEvent)
                 elseif isTrigger then
-                  -- if the event is requesting a trigger's updated be sent in to the custo trigger then check it isn't requesting itself then add to a table to
-                  -- be processed once all the Aura's triggers are added
                   local requestedTriggernum = tonumber(i)
                   if requestedTriggernum then
-                    if trigger_to_custom[requestedTriggernum] and trigger_to_custom[requestedTriggernum][triggernum] then
+                    if TTC_events[id] and TTC_events[id][triggernum] and TTC_events[id][triggernum][requestedTriggernum] then
                       -- if the request is reciprocal (2 custom triggers request each other which would cause a stack overflow) then neither get added.
-                      trigger_to_custom[requestedTriggernum][triggernum] = nil
                     elseif requestedTriggernum and requestedTriggernum ~= triggernum then
                       tinsert(trigger_events, "WA_TRIGGER_"..id.."_"..i)
```

Comment:
you can remove that line

Reply:
I don't think that's there any more already, not sure why it was showing. Gone now in any case.